ui/src/logs/components/expandable_message/ExpandableMessage.scss

```diff
@@ -77,3 +77,8 @@
     cursor: pointer;
   }
 }
+
+/* TODO: Make fancyscroll more customizable */
```

Comment:
should this comment still be here? or is that for later?

Reply:
It's should be here as a reminder. Once fancyscroll has a better way to hide the horizontal scrollbar this and the similar class in the reusable dropdown css can be removed.